index.js

```diff
@@ -0,0 +1,21 @@
+import Bookshelf from './modules/Bookshelf.js';
+import { startTime, toggleDisplay } from './modules/Navigation.js';
+
+const bookShelf = new Bookshelf();
+const addButton = document.getElementById('add-book-button');
+addButton.addEventListener('click', () => { bookShelf.addBook(); });
```

Comment:
[OPTIONAL] I think this form could use some improvements, for instance, you could listen for the event 'submit' instead of a click which would enable automatically the user to submit a new book with the key `enter`. [Read more about submit event here.](https://developer.mozilla.org/en-US/docs/Web/API/HTMLFormElement/submit_event)

Also, redirecting the user to the list of books after submitting a new book could be a great improvement and could help you provide a better UX.